### PR TITLE
Create bip-tkhan-CCVS.mediawiki

### PR DESCRIPTION
**Pull request by tenakakhan**

### CODE REVIEW

File: bip-tkhan-CCVS.mediawiki [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Layer: Process
+Title: Community Consensus Voting System
+Author: t.khan <teekhan42@gmail.com>
+Comments-Summary: No comments yet.
+Comments-URI: TBD
+Status: Draft
+Type: Standards Track
+Created: 2017-02-10
+License: BSD-2
+Voting Address: 3CoFA3JiK5wxe9ze2HoDGDTmZvkE5Uuwh8  (just an example, don’t send to this!)
```
Comment (luke-jr) [inline bip-tkhan-CCVS.mediawiki]: This header is invalid until the Process BIP is accepted.

File: bip-tkhan-CCVS.mediawiki [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Block #454,477 (~February 24, 2017): Final deadline for max block size BIP submissions for voting.
+
+Block #454,621 (~February 25, 2017): Voting officially begins.
+
+Block #455,629 (~March 4, 2017): Voting ends.
+
+==Implementation==
+
+Immediately: Announcement will be made on bitcoincore.org calling for any further max block size related BIP submissions by Block #454,477 (~February 24, 2017).
```
Comment (luke-jr) [inline bip-tkhan-CCVS.mediawiki]: bitcoincore.org is a private website outside the purview of the BIP process.

File: bip-tkhan-CCVS.mediawiki [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+Before Block #454,621: All max block size related BIPs are assigned a bitcoin address for voting. Voting begins. Announcement made on bitcoincore.org.
```
Comment (luke-jr) [inline bip-tkhan-CCVS.mediawiki]: Proposing voting on hardforks is in itself a hardfork, so this wouldn't be exclusively a Process BIP. Acceptance of this BIP would require approval from the entire Bitcoin community.

File: bip-tkhan-CCVS.mediawiki [hunk 4]
```diff
@@ -0,0 +1,82 @@
+<pre>
+BIP: ?
+Layer: Process
```
Comment (luke-jr) [inline bip-tkhan-CCVS.mediawiki]: "Process" is not a layer. This needs to be "Consensus (hard fork)"

**Comment by luke-jr** [comment]
P.S. I discourage even submitting this, as it has already been rejected, and as such it is a waste of everyone's time.

**Comment by btcdrak** [comment]
This proposal has been soundly rejected on the basis of being unworkable, so it really doesnt make sense to publish it.

**Comment by luke-jr** [comment]
@tenakakhan Please either address the issues or close this.

**Comment by MarcoFalke** [comment]
Should be closed

**Pull request closed without merging**